pkg/bundle/bundle.go

```diff
@@ -68,21 +69,29 @@ func PackAndPush(ref name.Reference, directory string, options ...remote.Option)
 // PullAndUnpack a container image layer content into a local directory. Analog
 // to the bundle.PackAndPush function, optional remote.Option can be used to
 // configure settings for the image pull, i.e. access credentials.
-func PullAndUnpack(ref name.Reference, targetPath string, options ...remote.Option) error {
+func PullAndUnpack(
+	ref name.Reference,
+	targetPath string,
+	options ...remote.Option,
+) (*v1.Image, error) {
```

Comment:
Thanks for the suggestion and I will update it.

TBH, I am not a big fan of long lines (it has `104` chars in a single line). It would be better if we consider having a lint rule for the line length.

Reply:
I must mention that I am a big fan of long lines. :-) 

Long lines allow me to see more logic because more horizontal lines without scrolling. If details matter, then I can look at the details (for example which variables get passed into a function call) of the longer line.

But just my few cents, within any project with more than one contributor, there'll rarely be a common opinion on those aspects. ;-)